Update installation; OpenSSL needs Root install

BenjaminHCCarr opened the pull request:
It took me a few runs to figure out why dependancies were failing at least on OSX, following the instructions, It took a bunch of log checking to figure out that openssl was failing, but was not obvious, as it would error on a different package that was an intermediary.

OpenSSL couldn't built without root because of the out-of-R tree it required.

BenjaminHCCarr committed:
Update installation; OpenSSL needs Root install

It took me a few runs to figure out why dependancies were failing at least on OSX, following the instructions, It took a bunch of log checking to figure out that openssl was failing, but was not obvious, as it would error on a different package that was an intermediary.

OpenSSL couldn't built without root because of the out-of-R tree it required.

flying-sheep commented:
hi, thanks for the info.

but this apparently hasn’t been true for me, so it’s probably platform specific.

maybe you could instead add the information into the `<aside/>` [here](https://github.com/IRkernel/irkernel.github.io/blob/3d084bb890d58fd56bda8f9710c446cf19ab77e0/installation/index.html#L95-L107)? (because that’s already about an OSX quirk)

>

BenjaminHCCarr committed:
Update SSL on OSX warning to an <aside>

Changed the OSX OpenSSL instructions to an aside.
May possibly need moving out of current block, but I believe it should be active here because of the tabs stanza.

BenjaminHCCarr commented:
@flying-sheep on linux it wouldn't apply, as it appears not to on windows. Maybe OSX and windows should be separated?
similar to: https://irkernel.github.io/installation/#source-panel
if the instructions are platform dependent as well

The `Source` title is a little misleading, as `apt-get` and `yum` are installing binary files, not pulling the raw source and compiling most of the code. It's just some aspects of the `Jupyter` code that are being recompiled.

flying-sheep commented on installation/index.html:
```diff
@@ -19,10 +19,28 @@
 	</div>
 	<div class="mdl-tabs__panel is-active" id=binary-panel>
+		<aside class="card-note mdl-card mdl-shadow--2dp">
+			<div class="mdl-card__title">
```
please leave out quotes when there’s only one class

flying-sheep commented on installation/index.html:
```diff
@@ -19,10 +19,28 @@
+				<pre class="r mdl-shadow--2dp"><code>>
+				user$ sudo R
+				> install.packages(c('openssl')
+				> quit()</code></pre>
```
please don’t add `quit()` and how to start an R interpreter. people know how to do that 😉

flying-sheep reviewed (changes requested):
could you please also show how it looks after the changes in a screenshot?

flying-sheep commented:
about your concerns for platform-specific instructions:

maybe we should just switch to a UI where people select their platform and if the want a non-standard install (e.g. dev versions), and only then show instructions

flying-sheep commented:
hi, why did you close this? it just needed a few changes to be merged

BenjaminHCCarr commented:
@flying-sheep oh, I was trying to clean up my laundry list of PR's, and didn't realize it was still of interest. But if it is still relevant, I will re-open, I had a huge project sent out and was trying to id things of interest still to upstream.

*reopen*

flying-sheep commented:
thank you! would you like to do the requested changes or should i just close this and make the edits manually?
